### PR TITLE
NEAR Plugin and Wrapper

### PR DESCRIPTION
**Pull request by ramilexe**

### CODE REVIEW

**Review by krisbitney — CHANGES_REQUESTED** [review]
It's so exciting to see this coming to fruition. I'm happy to see the test coverage has improved significantly since the original Near demo! 

I don't know the scope of your grant, so please take that into consideration when reading my questions.

File: protocol/near/plugin-js/schema.graphql [hunk 1]
```diff
@@ -48,6 +53,10 @@ type Mutation {
     meta: String
   ): Boolean!
 
+  createKey(
+    accountId: String!
+    networkId: String!
+  ): PublicKey!
   # sends a signed transaction and awaits execution
   sendTransaction(
```
[the remark this answers is not in the document]
Comment (Brioline) [reply protocol/near/plugin-js/schema.graphql:61]: resolved by https://github.com/polywrap/integrations/pull/46/commits/e3b327a3f4a41332814ee6c833b0394bb0a43291

[the remark this answers is not in the document]
Comment (ramilexe) [reply protocol/near/plugin-js/schema.graphql:61]: We didn't include migrating Borsch (which is required for serialization) to Wrapper in this scope, so we used it through Plugin. We have plan to apply to next grant to improve this and remove Near JS SDK as much as possible.

File: protocol/near/wrapper/src/utils/actionCreators.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { Near_AccessKey, Near_Action, Near_PublicKey } from "../query/w3";
+
+import { BigInt, JSON } from "@web3api/wasm-as";
+import * as bs64 from "as-base64";
```
[the remark this answers is not in the document]
Comment (Brioline) [reply protocol/near/wrapper/src/utils/actionCreators.ts]: resolved by https://github.com/polywrap/integrations/pull/46/commits/e3b327a3f4a41332814ee6c833b0394bb0a43291

File: protocol/near/plugin-js/schema.graphql (hunk 1 above)
[the remark this answers is not in the document]
Comment (krisbitney) [reply protocol/near/plugin-js/schema.graphql:61]: Okay, I see. Cool stuff!

**Review by krisbitney — APPROVED** [review]

**Review by dOrgJelli — APPROVED** [review]
All LGTM! Amazing work 👏

**Pull request merged**